docs/backup-restore.md

```diff
@@ -1,134 +1,208 @@
+  <button class="tablinks" onclick="openTab(event, 'PostgreSQL1', 'tabset-1')">PostgreSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'SQLite1', 'tabset-1')">SQLite</button>
+</div>
+
+<div id="Amazon_RDS_or_Azure_Database_for_MySQL_or_PostgreSQL1" class="tabcontent" markdown="1">
+
+You can restore to any point within the backup retention period by using the automated backup feature.
+</div>
+<div id="MySQL1" class="tabcontent" markdown="1">
+
+Use the `mysqldump` command with the `--single-transaction` option.
+</div>
+<div id="PostgreSQL1" class="tabcontent" markdown="1">
 
 Use the `pg_dump` command.
+</div>
+<div id="SQLite1" class="tabcontent" markdown="1">
+
+Use the `.backup` command with the `.timeout` command as specified in [Special commands to sqlite3 (dot-commands)](https://www.sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_)
+
+For an example, see [BASH: SQLite3 .backup command](https://stackoverflow.com/questions/23164445/bash-sqlite3-backup-command).
+</div>
+</div>
+
+### Back up with explicit pausing
+
+Another way to create a transactionally consistent backup is to create a backup while ScalarDB Cluster does not have any outstanding transactions. Creating the backup depends on the following:
+
+- If the underlying database has a point-in-time snapshot or backup feature, you can create a backup during the period when no outstanding transactions exist.
+- If the underlying database has a point-in-time restore or recovery (PITR) feature, you can set a restore point to a time (preferably the mid-time) in the pause duration period when no outstanding transactions exist.
+
+{% capture notice--info %}
+**Note**
+
+When using a PITR feature, you should minimize the clock drifts between clients and servers by using clock synchronization, such as NTP. Otherwise, the time you get as the paused duration might be too different from the time in which the pause was actually conducted, which could restore the backup to a point where ongoing transactions exist.
 
-#### SQLite (backup)
+In addition, you should pause for a sufficient amount of time (for example, 10 seconds) and use the mid-time of the paused duration as a restore point since clock synchronization cannot perfectly synchronize clocks between nodes.
+{% endcapture %}
 
-Use the [`.backup` command](https://www.sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_) along with `.timeout`.
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-See also [this post](https://stackoverflow.com/questions/23164445/bash-sqlite3-backup-command), for example.
+To make ScalarDB drain outstanding requests and stop accepting new requests so that a pause duration can be created, you should use ScalarDB Cluster, which uses the Scalar Admin interface, or implement the [Scalar Admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application that uses ScalarDB.
```

Comment:
I've just realized ScalarDB Cluster is mentioned here as well. As I pointed out earlier, ScalarDB Cluster is not included in the community version. We should make a note of that.

Reply:
Got it. I've added notes on both mentions of ScalarDB Cluster.

Reply:
Thanks! Should we also add a link to the ScalarDB Cluster documentation?

Reply:
I had considered that, but readers of the Community docs wouldn't be able to access the ScalarDB Cluster documentation since it's only available on the Enterprise docs site.

Also, if we add a static link, we would only be able to add one for the `latest` version (which would be confusing for users who are looking at docs from a previous release).

Reply:
Understood. We need to discuss how we reference ScalarDB Cluster in the Community docs.

@feeblefakie what are your thoughts on this?

Reply:
@brfrn169 (cc: @feeblefakie) Regarding the following part, I was incorrect, so let me rephrase:

> Also, if we add a static link, we would only be able to add one for the `latest` version (which would be confusing for users who are looking at docs from a previous release).

We could add static links that point to a ScalarDB Cluster document for this doc on branches `3.9` and `3.10`. 

However, I'm still concerned that Community users might be confused if they click the link and find themselves going from the Community docs site to the Enterprise docs site. If that's a concern we're willing to make an exception for, I'm OK with that.

Reply:
@josh-wong 
Can we do something like this to let users to notice that they are going to the enterprise doc site?

[ScalarDB Cluster (going to Enterprise docs site)](link)

Reply:
@feeblefakie
I'm OK using "[ScalarDB Cluster (redirects to the Enterprise docs site)](#)". 

@brfrn169 I'll add a link to the page for the version that the user is on when backporting this PR. For this PR, which is for the `master` branch, I'll specify `latest` as the link.